### ISSUE TITLE
[zglfw] getProcAddress is not wrapped

### ISSUE THREAD

**Issue by TechPersonYT** [michal-z/zig-gamedev#360]
This minimal OpenGL/GLFW example compiles, but does not run with `zglfw` and `zopengl`:

```zig
pub fn main() !void {
    try zglfw.init();
    defer zglfw.terminate();

    var window = try zglfw.Window.create(1280, 720, "Test Window", null);

    zglfw.makeContextCurrent(window);

    while (!window.shouldClose()) {
        gl.clear(gl.COLOR_BUFFER_BIT);

        window.swapBuffers();

        zglfw.pollEvents();
    }
}
```

Specifically, it segfaults on the call to `gl.clear`. My guess is that `zopengl` wants a function pointer, which I would normally retrieve from `glfw.getProcAddress`. The [README](https://github.com/michal-z/zig-gamedev/blob/main/libs/zopengl/README.md) for `zopengl` indicates that such a function should exist for both the SDL and GLFW bindings, but so far I've only been able to find the former.

**Comment by michal-z** [comment]
Yes, you are right. `zglfw` currently does not support OpenGL. I will try to fix this.

For now you could use `zsdl` with OpenGL - we have nice minimal example for this: https://github.com/michal-z/zig-gamedev/blob/main/samples/minimal_gl/src/minimal_gl.zig

**Comment by TechPersonYT** [comment]
Thanks for the prompt response-- I might wait for GLFW support since it's a bit lighter weight and I'm more familiar with it. It may also be useful to have a minimal GLFW/OpenGL example, though this could require some renaming. If that sounds feasible I'm basically writing a port of the SDL example code already.

**Comment by hazeycode** [comment]
done: https://github.com/michal-z/zig-gamedev/pull/395
